lib/cc/cli/init.rb

```diff
@@ -10,7 +10,7 @@ class Init < Command
       def run
         if !upgrade? && filesystem.exist?(CODECLIMATE_YAML)
           warn "Config file .codeclimate.yml already present.\nTry running 'validate-config' to check configuration."
-          create_default_configs
+          create_default_engine_configs
```

Comment:
They're in separate commits to make review easier, if that helps. I can extract the rename commit as a separate PR if you'd like.